tests/yam/agama/boot_agama.pm

```diff
@@ -22,6 +23,18 @@ use bootloader_s390;
 use bootloader_zkvm;
 use bootloader_pvm;
 
+sub read_agama_package {
```

Comment:
wasn't a PRE value somewhere? setting the var to 17 how does it helps if production and development would have the same? parsing further the package version I also don't know how it would be practical. I thought we agreed that the other value was more interesting and this one was something only to display.

in the future we might need to create a condition is_agama(<=17) and first will need to compare the numeric value and then the presence of the PRE (non-released) so whatever we store in the variable should useful for that.